velox/functions/lib/Re2Functions.h

```diff
@@ -26,6 +26,22 @@
 
 namespace facebook::velox::functions {
 
+/// Representation of different kinds of patterns.
+enum class PatternKind {
+  /// Pattern containing wildcard character '_' only.
+  kWildcardPatternOne,
```

Comment:
We can shorten these names by removing Pattern. Since these always using with PatternKind:: prefix, having Pattern in the name is redundant.